graph/src/components/subgraph/proof_of_indexing/event.rs

```diff
@@ -62,6 +66,7 @@ impl fmt::Debug for ProofOfIndexingEvent<'_> {
                 builder.field("id", id);
                 builder.field("data", &data.iter().collect::<BTreeMap<_, _>>());
             }
+            Self::DeterministicError => {} // NOOP
```

Comment:
Is something still written to the PoI, even though this says 'noop'?

Reply:
Yes, the discriminant is written above.